2025-09-05 15:32:14,240 - large_scale_cognitive_test_client_7 - INFO - Starting client 7 with config: {'client_id': 7, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 463, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:32:18,148 - large_scale_cognitive_test_client_7 - INFO - Client 7 initialized with 6176 training samples
2025-09-05 15:32:18,148 - large_scale_cognitive_test_client_7 - INFO - Client 7 connecting to 0.0.0.0:8080
2025-09-05 15:32:18,148 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:32:18,163 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:32:18,164 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:32:18,165 - flwr - DEBUG - ChannelConnectivity.CONNECTING
2025-09-05 15:32:18,165 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:32:18,372 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:32:18,372 - large_scale_cognitive_test_client_7 - ERROR - Client 7 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_status:14, grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"}"
>
2025-09-05 15:38:31,155 - large_scale_cognitive_test_client_7 - INFO - Starting client 7 with config: {'client_id': 7, 'server_address': '0.0.0.0:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 463, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:38:35,142 - large_scale_cognitive_test_client_7 - INFO - Client 7 initialized with 6176 training samples
2025-09-05 15:38:35,142 - large_scale_cognitive_test_client_7 - INFO - Client 7 connecting to 0.0.0.0:8080
2025-09-05 15:38:35,142 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:38:35,160 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:38:35,161 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:38:35,162 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:38:35,368 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:38:35,369 - large_scale_cognitive_test_client_7 - ERROR - Client 7 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:0.0.0.0:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
2025-09-05 15:41:16,712 - large_scale_cognitive_test_client_7 - INFO - Starting client 7 with config: {'client_id': 7, 'server_address': '20.169.237.127:8080', 'experiment_name': 'large_scale_cognitive_test', 'seed': 463, 'batch_size': 32, 'epochs': 2, 'learning_rate': 0.001, 'attack_enabled': False}
2025-09-05 15:41:20,591 - large_scale_cognitive_test_client_7 - INFO - Client 7 initialized with 6176 training samples
2025-09-05 15:41:20,591 - large_scale_cognitive_test_client_7 - INFO - Client 7 connecting to 20.169.237.127:8080
2025-09-05 15:41:20,591 - flwr - WARNING - DEPRECATED FEATURE: flwr.client.start_client() is deprecated.
	Instead, use the `flower-supernode` CLI command to start a SuperNode as shown below:

		$ flower-supernode --insecure --superlink='<IP>:<PORT>'

	To view all available options, run:

		$ flower-supernode --help

	Using `start_client()` is deprecated.

            This is a deprecated feature. It will be removed
            entirely in future versions of Flower.
        
2025-09-05 15:41:20,606 - flwr - DEBUG - Opened insecure gRPC connection (no certificates were passed)
2025-09-05 15:41:20,606 - flwr - DEBUG - ChannelConnectivity.IDLE
2025-09-05 15:41:20,607 - flwr - DEBUG - ChannelConnectivity.CONNECTING
2025-09-05 15:41:20,961 - flwr - DEBUG - ChannelConnectivity.TRANSIENT_FAILURE
2025-09-05 15:41:21,168 - flwr - DEBUG - gRPC channel closed
2025-09-05 15:41:21,169 - large_scale_cognitive_test_client_7 - ERROR - Client 7 failed: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)"
	debug_error_string = "UNKNOWN:Error received from peer  {grpc_message:"failed to connect to all addresses; last error: UNKNOWN: ipv4:20.169.237.127:8080: Failed to connect to remote host: connect: Connection refused (61)", grpc_status:14}"
>
